dags/atd_service_bot_index_issues_to_dts_portal.py

```diff
@@ -1,5 +1,5 @@
 import os
-from datetime import datetime, timedelta
+import pendulum
```

Comment:
@frankhereford this is super minor, but since we're laying down a pattern that will be copied from here on out: i think we will save tiny bits of RAM on the DAG preprocessor if you target the pendulum module imports:

```python
from pendulum import datetime, duration
```

Reply:
Great idea John, I'll do that. Let's make our airflow setup as lean and efficient as we can!